src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,58 @@
+    },
+    {
+      sideBarItemViewBox: "0 0 28 20",
+      sideBarItemText: "Preview",
+      sideBarItemLink:
+        "M14 3C18.5248 3 22.8894 7.78 24.6513 10C22.8894 12.22 18.5248 17 14 17C9.47521 17 5.1106 12.22 3.34874 10C5.1106 7.78 9.47521 3 14 3ZM14 0C7.73337 0 2.38772 6.16 0.385601 8.82C0.13504 9.16253 0 9.57577 0 10C0 10.4242 0.13504 10.8375 0.385601 11.18C2.38772 13.84 7.73337 20 14 20C20.2666 20 25.6123 13.84 27.6144 11.18C27.865 10.8375 28 10.4242 28 10C28 9.57577 27.865 9.16253 27.6144 8.82C25.6123 6.16 20.2666 0 14 0ZM14 5C13.2566 4.99672 12.5227 5.16799 11.8577 5.5C12.3053 5.52444 12.7353 5.68232 13.0922 5.95326C13.449 6.2242 13.7164 6.59576 13.8599 7.02C13.991 7.41631 14.0097 7.84114 13.9139 8.24741C13.8181 8.65369 13.6115 9.02552 13.3171 9.3217C13.0227 9.61787 12.6519 9.82681 12.2459 9.92536C11.8399 10.0239 11.4145 10.0082 11.0168 9.88C10.5922 9.73674 10.2202 9.46962 9.94897 9.11312C9.67775 8.75662 9.5197 8.32711 9.49524 7.88C9.16591 8.53827 8.99454 9.2641 8.99471 10C8.99471 10.9889 9.28826 11.9556 9.83825 12.7779C10.3882 13.6001 11.17 14.241 12.0846 14.6194C12.9992 14.9978 14.0056 15.0969 14.9765 14.9039C15.9474 14.711 16.8393 14.2348 17.5393 13.5355C18.2393 12.8363 18.716 11.9454 18.9091 10.9755C19.1022 10.0055 19.0031 9.00021 18.6243 8.08658C18.2454 7.17295 17.6039 6.39206 16.7808 5.84265C15.9577 5.29325 14.99 5 14 5Z",
+    },
+  ];
+  return (
+    <Box
+      bg="#2C3333"
+      w="98px"
+      h="960px"
+      color="#C8C5C5"
+      fontWeight="700"
+      position="absolute"
+      top="100px"
+      left="0"
+    >
+      <Flex align="center" justify="center" flexDirection="column" maxW="98px">
+        {sideBarContent.map((item, index) => (
+          <SideBarItem
+            key={index}
```

Comment:
You can check out this article -> https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/